ui/src/dms/MessagesSidebarItem.tsx

```diff
@@ -36,14 +40,7 @@ function ChannelSidebarItem({ whom, brief }: MessagesSidebarItemProps) {
     <SidebarItem
       to={`/groups/${groupFlag}/channels/${whom}`}
```

Comment:
```suggestion
      to={`/groups/${groupFlag}/channels/${nest}`}
```

If this is set to whom it won't route to the actual channel.